core/src/main/scala/kafka/docker/KafkaDockerWrapper.scala

```diff
@@ -87,8 +87,12 @@ object KafkaDockerWrapper {
     parser.parseArgsOrFail(args)
   }
 
-  private def formatStorageCmd(configsPath: Path, env: Map[String, String]): Array[String] = {
-    Array("format", "--cluster-id=" + env.get("CLUSTER_ID"), "-c", s"${configsPath.toString}/server.properties")
+  private[docker] def formatStorageCmd(configsPath: Path, env: Map[String, String]): Array[String] = {
+    val clusterId = env.get("CLUSTER_ID") match {
+      case Some(str) => str
+      case None => throw new RuntimeException("CLUSTER_ID environment variable is not set.")
```

Comment:
Could you add a test for this in `testFormatStorageCmd` ? `assertThrows...`

Reply:
Sure, I just force-pushed a commit that adds this check.